docs/04-guides/on-demand/01-upload-video-asset/02-api.mdx

```diff
@@ -208,3 +210,101 @@ curl --location --request GET 'https://livepeer.studio/api/asset/{id}' \
 
 </CodeBlockTabItem>
 </CodeBlockTabs>
+
+## Resumable Upload
+
+The resumable upload method allows a user to resume their upload if their 
+connection fails due to a cancelled request or a dropped connection. If you're
+uploading a large file and you lose connection at 98%, you don't have to restart
+from the beginning, you can continue back at 98%.
+
+### Step 1: Generate upload URL
+
+To upload the asset to the Livepeer network locally, you'll need to make a POST
+request to generate an upload URL.
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="Node" label="Node">
+
+```js
+const uploadAsset = await fetch(
+  "https://livepeer.studio/api/asset/request-upload",
+  {
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${process.env.API_TOKEN}`,
+      "Content-Type": "application/json",
+    },
+    data: JSON.stringify({
+      name: "Example name",
+    }),
+  }
+);
+```
+
+</CodeBlockTabItem>
+<CodeBlockTabItem value="curl" label="curl">
+
+```bash
+curl --location --request POST 'https://livepeer.studio/api/asset/request-upload' \
+--header 'Authorization: Bearer $API_TOKEN' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "name":"Example name"
+}'
+```
+
+</CodeBlockTabItem>
+</CodeBlockTabs>
+
+### Step 2: Upload your resumable video via Tus
+
+Livepeer Studio supports resumable uploads via [Tus](https://tus.io/). This section provides a simple 
+example of how to use `tus-js-client` to upload a video file.
+
+You should use the `tusEndpoint` field of the [response object](/reference/api/create-asset#response-example) 
+to upload the video file and track the progress:
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="Node" label="Node">
+
+```js
+//  This assumes there is an `input` element of `type="file"` with id `fileInput` in the HTML
+const input = document.getElementById("fileInput");
```

Comment:
You're instructing the user to access a DOM element in the browser (client side) but the code block is labeled Node (server side).

Reply:
Thanks for catching this

Reply:
I think it's confusing that we tell users to use a mix of server and client side code. The other code blocks are labeled Node. Let's be consistent across all code blocks. Either we assume everything is written on the client or the server.

Reply:
I think we should just rename all `Node` codeblocks to Javascript. What do you think?